backend/internal_api/src/main.rs

```diff
@@ -41,11 +41,12 @@ async fn main() -> io::Result<()> {
     db.seed_data().await;
 
     let host = config.host.clone();
+		info!("Running backend at {}:{}", config.host, config.port); // log server start
 
     // start server at HOST:PORT, persisting Db connection
     HttpServer::new(move || {
-        
-        let cors = Cors::permissive();
+
+				let cors = Cors::permissive();
```

Comment:
Look's like its just an indentation change.

Reply:
I changed the indentation to match main. I think it should be good now.